web/yo/app/scripts/controllers/gene.js

```diff
@@ -2464,7 +2306,7 @@ angular.module('oncokbApp')
             $scope.specifyAnnotation = function() {
                 var annotationLocation = {};
                 setAnnotationResult(annotationLocation, fetchResults(FindRegex.result(this.gene.background.text)), 'Gene Background');
-                var mutations = stringUtils.getGeneData(this.gene, false, false, false, false, true).mutations;
+                var mutations = stringUtils.getGeneData(this.gene, true, false).mutations;
```

Comment:
We only removed `excludeObsolete` and `excludeRedHands`, but why there are three parameters been removed here?

Reply:
Previously, getGeneData() took 5 parameters and it was a mistake to pass 6